index.html

```diff
@@ -34,10 +34,11 @@ <h2 class="tel"><a href="tel:+380990671174">+38(099) 067-11-74</a></h2>
         </div>
     </header>
     <main>
-        <div class="work">
-            <h3 class="text-about-us">
-                Замовте майстра на всі руки, якому можна довіряти</h3>
-        </div>
+        <div class="hero-section">
+            <h3 class="motto">
+              Замовте майстра на всі руки, якому можна довіряти!<br>
+              Ваше задоволення - наш пріоритет.</h3>
+         </div>
         <h1 class="main-title">РOБОТИ, ЯКІ МИ ВИКОНАЄМО ДЛЯ ВАС:</h1>
```

Comment:
Please rewrite the text as a usual sentence, with no caps-lock. You can use a css rule to make all characters capitalised. Also, make sure the top margin and bottom one are equal.